tests/tck/features/match/MatchById.feature

```diff
@@ -973,3 +973,68 @@ Feature: Match By Id
       | ("Warriors" :team{name: "Warriors"})                                                                        |
       | ("Hornets" :team{name: "Hornets"})                                                                          |
       | ("Spurs" :team{name: "Spurs"})                                                                              |
+
+  Scenario: match by multiple id
+    When executing query:
+      """
+      MATCH (a)-[e:like]->(b)
+      WHERE id(a) == 'Tim Duncan' OR id(b) == 'Tony Parker'
```

Comment:
Should add integer vid test case too.

Reply:
ok， add case in next pr

Reply:
Why don't add in this PR directly?

Reply:
@Sophie-Xie  waiting for this PR to publish test version